apis/Google.Cloud.AspNetCore.DataProtection.SecretManager/IntegrationTests/SecretManagerFixture.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2023 Google LLC
```

Comment:
Please rename IntegrationTests to something like DataProtection.SecretManager.IntegrationTests; the project generator looks for a suffix of ".IntegrationTests.csproj" so it needs the period.

I'd then use a *namespace* which matches our normal convention - so Google.Cloud.AspNetCore.DataProtection.SecretManager. It's only the directory/file name that needs to be unconventional.